FE/src/pages/Driver/PickUp/SelectChild/TodayPickUpList.jsx

```diff
@@ -0,0 +1,50 @@
+import { useNavigate } from "react-router-dom";
+import styles from "./ContentsBox.module.scss";
+import iDrop from "@/assets/iDropGreen.svg";
+
+export function TodayPickUpList({ childrenData }) {
+    const navigator = useNavigate();
+    const movePage = (index) => {
+        navigator("/pickup", { state: childrenData[index] });
+    };
+
+    const render = childrenData.map(
+        (
+            {
+                childId,
+                childImage,
+                childName,
+                pickUpStartTime,
+                pickUpEndTime,
+                startAddress,
+                endAddress
+            },
+            index
+        ) => {
+            return (
+                <div key={index} className={styles.content}>
+                    <img
+                        className={styles.kidImg}
+                        src={childImage || iDrop}
+                    ></img>
+                    <div className={styles.infoBox}>
+                        <div>
+                            <span> {childName} </span>
+                            <button onClick={() => movePage(index)}>
```

Comment:
```suggestion
<button onClick={() => movePage(childId)}>
```
이거 params로 요청하는거면 childId로 바꿔야할 것 같아요